WordOfTheDay.Repository/WordsRepository.cs

```diff
@@ -47,12 +43,12 @@ public async Task<List<WordCount>> CloseWords(string email)
                 predicate = predicate.Or(
                     closeWord => EF.Functions.Like(closeWord.Text, key) 
                     && closeWord.Text.Length <= word.Length + 1 
-                    && closeWord.Text != word
-                    && closeWord.AddTime > time);
+                    && closeWord.Text != word);
             }
 
             var closeWords = _context.Words
                 .AsExpandable().Where(predicate)
```

Comment:
Let's move Where statemenet to extension method, e.g. ".FilterClosestWords()". Also move all code that generates predicate there